### PR TITLE
add first components and test 1of4

### PR DESCRIPTION
**Pull request by TmWgnr**

### CODE REVIEW

File: components/ArtPiecePreview/index.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import Image from "next/image";
+
+export default function ArtPiecePreview({ slug, image, title, artist }) {
+  console.log(artist);
```
Comment (ghost) [inline components/ArtPiecePreview/index.js:4]: Remember to delete/ comment all console.logs before pushing the code to remote!

Comment (sebastianwel) [reply components/ArtPiecePreview/index.js:4]: good point! thanks 👍

File: components/ArtPieces/index.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import Image from "next/image";
```
Comment (ghost) [inline components/ArtPieces/index.js:1]: As I can see, this Import is not needed and can be deleted, right?

File: components/ArtPieces/index.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+export default function ArtPieces({ pieces }) {
+  return (
+    <section>
+      <ul>
+        {!pieces ? (
+          <h1>...loading</h1>
+        ) : (
+          pieces.map((piece) => {
+            console.log(piece.artist);
```
Comment (ghost) [inline components/ArtPieces/index.js:12]: Another console.log to comment

File: pages/index.js [hunk 4]
```diff
@@ -1,7 +1,14 @@
+import useSWR from "swr";
+import ArtPieces from "@/components/ArtPieces";
+
+const URL = "https://example-apis.vercel.app/api/art";
+
 export default function HomePage() {
+  const { data } = useSWR(URL);
+  console.log(data);
```
Comment (ghost) [inline pages/index.js:8]: Another console.log to comment

**Review by ghost — APPROVED** [review]
Very clean code, pretty easy to read, nice 👍
I only left a few comments, but besides the console .logs its nothing really important.
To explain, why we want the console.logs commented is, because if we don't pay attention we might leak sensible data to the user which could cause a security problem for our website. If we have e.g a user login etc.

**Pull request merged**